Channels/UserInterface/02c_SemiAutomaticMapping.md

```diff
@@ -0,0 +1,94 @@
+[!!Manage the offers](../Operation/01_ManageOffers.md)
+
+# Semi automatic mapping
+
+*Omni-Channel > Offers > Tab SEMI AUTOMATIC MAPPING*
+
+![Semi automatic mapping](../../Assets/Screenshots/Channels/Offers/SemiAutomaticMapping/SemiAutomaticMapping.png "[Semi automatic mapping]")
+
+**Semi-automatic changes awaiting confirmation**
+
+- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   
+  Click this button to update the list of offers awaiting confirmation.
+
+- *View*   
+  Click the drop-down list to select the view. All created views are displayed in the drop-down list. Click the ![Points](../../Assets/Icons/Points01.png "[Points]") (Points) button to the right of the *View* drop-down list to display the context menu and create or edit a view.    
+
+  - View context menu   
+    Click the ![Points](../../Assets/Icons/Points01.png "[Points]") (Points) button to the right of the *View* drop-down list to display the context menu. The following menu entries are available:
+
+      - ![Create](../../Assets/Icons/Plus06.png "[Create]") create   
+        Click this entry to create a view. The *Create view* window is displayed.
+
+      - ![Rename](../../Assets/Icons/Edit02.png "[Rename]") rename    
+        Click this entry to rename the selected view. The *Rename view* window is displayed. This menu entry is only displayed if a view has been selected.
+
+      - ![Reset](../../Assets/Icons/Reset.png "[Reset]") reset    
+        Click this entry to reset all unsaved changes to the settings of the selected view. This menu entry is only displayed if a view has been selected and any changes have been made to the view settings.
+
+      - ![Publish](../../Assets/Icons/Publish.png "[Publish]") publish    
+        Click this entry to publish the view. This menu entry is only displayed if a view has been selected and unpublished.
+
+      - ![Private view](../../Assets/Icons/Unpublish.png "[Unpublish]") private view    
+        Click this entry to unpublish the view. This menu entry is only displayed if a view has been selected and published.
+
+      - ![Save](../../Assets/Icons/Save.png "[Save]") save    
+        Click this entry to save the current view settings in the selected view. This menu entry is only displayed if a view has been selected.
+
+        > [Info] When the settings of a view have been changed, an asterisk is displayed next to the view name. The asterisk is hidden as soon as the changes have been saved.
+
+      - ![Delete](../../Assets/Icons/Trash01.png "[Delete]") delete    
+        Click this entry to delete the selected view. A confirmation window is displayed to confirm the deletion. This menu entry is only displayed if a view has been selected.
+
+
+- ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") *List changes that cannot be confirmed by you*    
```

Comment:
Ansprache an Benutzer: "by you" oder "by the current user/(currently) logged in user?